Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -521,7 +558,7 @@ def run_audit(formula, alias_rename, old_contents)
           end
           return
         end
-        FileUtils.mv alias_rename.first, alias_rename.last if alias_rename.present?
+        FileUtils.mv T.must(alias_rename.first), T.must(alias_rename.last) if alias_rename.present?
```

Comment:
Removing the `T.must`s here is confusing me, because we already check `if alias_rename.present?` so the `.first` and `.last` shouldn't be `nil`. But I guess Sorbet doesn't "see" that. There's the same pattern in other places where I did too many `T.must`s, too. So I'm confused how we do this, because it's not "return/raise if it's nil", we want things to continue, just don't want to run the `FileUtils.mv`.

Reply:
This would be the Sorbet-friendly way to do this.

```suggestion
        if alias_rename && (source = alias_rename.first) && (destination = alias_rename.last)
          FileUtils.mv source, destination
        end
```

Sorbet can infer that `alias_rename` is not `nil` from `.present?` but not that `alias_rename.first` is not `nil`

Reply:
I wonder if in the future this could be improved using [`tuples`](https://sorbet.org/docs/tuples). Once they have better support, of course. But it seems like we could say that `alias_rename` is either `nil` or a two-element array so this works as expected...

Dreams for the future though!